docs/extensions/services/accessing-database.md

```diff
@@ -0,0 +1,72 @@
+---
+description: Learn about the ItemsService in Directus and how to utilize them when building extensions.
+contributors: Esther Agbaje
+---
+
+# Accessing Items in the Database
+
+The `ItemsService` allows you to work with and retrieve items data by creating endpoints to items in a collection. It
+provides a way to perform CRUD operations on items in a database.
+
```

Comment:
There's a bit missing here which is just talking about the initialization. It should cover the fact that the service requires a collection and a schema. And that you must import and use `getSchema()`.

```js
export default defineEndpoint((router, context) => {
  const { services, getSchema } = context;
  const { ItemsService } = services;

  router.get('/', async (req, res) => {
    const schema = await getSchema();
    const service = new ItemsService('collection_name', { schema });
  });
});
```

By doing this, the other examples are contextualized more. This is a simple example with a hardcoded collection and just a `/` route handler. For this top-of-file example, let's keep it as easy as possible. 

Questions:
1. Can `getSchema()` be called at the top level of the endpoint or must it be in the route handler?  Is there a standard best practice? If anything goes and there's no benefit to putting it in the route handler I'd move it up to remove repetition if there are multiple endpoints. 
2. What else can go in the options object other than schema? Are they worth mentioning? Accountability comes to mind?

Reply:
`getSchema()` can be called at the top level. Moved it there.

Accountabilility doesn't show up as a valid type. See screenshot
<img width="672" alt="Screenshot 2023-11-20 at 19 54 22" src="https://github.com/directus/directus/assets/53586167/6bc640fa-5176-4de2-a9e9-d981947c7b2e">

When the type is fixed, I'll circle back and update it accordingly.